libs/emoji.js

```diff
@@ -1179,33 +1132,29 @@ var emoji = (function() {
       return new Promise(function(resolve, reject) {
         var promises = [];
 
-        if (!config.customEmojiImageRegex) {
-          for (var i = 0; i < 14; i++) {
-            if (i === 11) {
-              continue;
-            }
-
-            images[i] = new Image();
-            images[i].src = "style/emoji/emoji" + i + ".png";
+        for (var i = 0; i < 13; i++) {
+          if (i == 11) {
+            continue;
+          }
 
-            promises.push(new Promise(function(resolve, reject) {
-              images[i].onload = resolve;
-            }));
+          images[i] = new Image();
+          var num = i.toString(16);
+          if (config.customEmojiImageFormat) {
+            var fileName = config.customEmojiImageFormat.replace("NUM", num);
```

Comment:
We should document *config.customEmojiImageFormat* in config/default.js, even if we don't define it there.